### DISCUSSION TITLE
Running with the latest version of Expo Go

### DISCUSSION THREAD

**Discussion by andyslucky** [victorbalssa/abacus#284] [Q&A] [answered]
Has anyone tried to run this project from scratch on the latest version of Expo Go? I get a failure due to SDK incompatibility. 
After updating expo following the instructions posted here [Upgrading Expo SDK Walkthrough](https://docs.expo.dev/workflow/upgrading-expo-sdk-walkthrough/) I did get the application to load in Expo Go, but I of course got some react native error upon launch:
```
...DebuggingOverlayNativeComponent.js: Unsupported param type for method "highlightTraceUpdates", param "updates". Found $ReadOnlyArray
```
I am not super familiar with React-Native projects, so any help would be appreciated.
Development Environment:
Windows 10 Pro (x64)
Node: v20.13.1
NPM: 10.5.2
Test Device: iPhone 14
Expo Go: Client version 1017614; Supported SDKs 51

**Comment by victorbalssa** [comment, marked as answer]
Hey I will update it to sdk 51 👍🏻

**Reply by andyslucky** [reply]
Thanks so much.
